lib/repl.js

```diff
@@ -150,7 +150,6 @@ function REPLServer(prompt,
   self.ignoreUndefined = !!ignoreUndefined;
   self.replMode = replMode || exports.REPL_MODE_SLOPPY;
   self.underscoreAssigned = false;
-  self.last = undefined;
```

Comment:
I have to admit I don’t see how this is related to the rest of the changes here?